mithril-aggregator/src/artifact_builder/cardano_database_artifacts/digest.rs

```diff
@@ -217,9 +296,19 @@ mod tests {
         let uploaders: Vec<Arc<dyn DigestFileUploader>> =
             vec![Arc::new(first_uploader), Arc::new(second_uploader)];
 
-        let builder = DigestArtifactBuilder::new(uploaders, TestLogger::stdout()).unwrap();
+        let builder = DigestArtifactBuilder::new(
+            Url::parse("https://aggregator/").unwrap(),
+            uploaders,
+            PathBuf::from("/tmp/mithril"),
```

Comment:
Maybe we want to use the temp directory function for tests here and below instead of `/tmp/mithril` to avoid tests collisions?

Reply:
I will create a dedicated temp folder for tests that call `upload` and just rename `/tmp/mithril` to `/tmp/whatever` for others that don't use this value

Reply:
Done